Source/WebCore/rendering/RenderThemeIOS.mm

```diff
@@ -723,12 +723,14 @@ static bool renderThemePaintSwitchTrack(OptionSet<ControlStyle::State>, const Re
 
 void RenderThemeIOS::adjustSwitchStyle(RenderStyle& style, const Element* element) const
 {
-    if (!style.width().isAuto() && !style.height().isAuto())
-        return;
+    // FIXME: Deduplicate sizing with the generic code somehow.
+    if (style.width().isAuto() || style.height().isAuto()) {
+        auto size = std::max(style.computedFontSize(), logicalSwitchHeight);
+        style.setLogicalWidth({ size * (logicalSwitchWidth / logicalSwitchHeight), LengthType::Fixed });
+        style.setLogicalHeight({ size, LengthType::Fixed });
+    }
 
-    auto size = std::max(style.computedFontSize(), logicalSwitchHeight);
-    style.setLogicalWidth({ size * (logicalSwitchWidth / logicalSwitchHeight), LengthType::Fixed });
-    style.setLogicalHeight({ size, LengthType::Fixed });
+    adjustSwitchStyleDisplay(style);
```

Comment:
Is it OK that we run the code after this line for `!style.width().isAuto() && !style.height().isAuto()` ? Seems harmless but just want to be sure.

Reply:
Yeah, not running it was a bug.